dali/operators/image/resize/resize_attr.cc

```diff
@@ -172,19 +121,19 @@ void ResizeAttr::ParseLayout(
   first_spatial_dim = spatial_dims_begin;
 }
 
-void ResizeAttr::CalculateInputRoI(SmallVector<float, 3> &in_lo,
-                                   SmallVector<float, 3> &in_hi,
-                                   const TensorListShape<> &input_shape,
-                                   int sample_idx) const {
-  in_lo.resize(spatial_ndim_);
-  in_hi.resize(spatial_ndim_);
+void CalculateInputRoI(SmallVector<float, 3> &in_lo, SmallVector<float, 3> &in_hi, bool has_roi,
+                       bool roi_relative, const float *roi_start, const float *roi_end,
```

Comment:
Kind of a nitpick, but how about using th span instead of roi_end, roi_start being plain pointers?